src/theme-variant-context.tsx

```diff
@@ -1,11 +1,12 @@
 'use client';
 import * as React from 'react';
 
-export type Variant = 'default' | 'inverse' | 'alternative';
+export type Variant = 'default' | 'inverse' | 'alternative' | 'overMedia';
```

Comment:
I think it's fine, but a bit weird having `'overMedia'` and not having `'overInverse'` or `'overAlternative'`

Reply:
Discussed offline. We decided to leave `overMedia` instead of media because we don't think there's much of a difference